composer.json

```diff
@@ -19,17 +19,23 @@
         "slack": "https://zendframework-slack.herokuapp.com",
         "forum": "https://discourse.zendframework.com/c/questions/expressive"
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/webimpress/zend-expressive-router"
+        }
+    ],
     "require": {
         "php": "^7.1",
         "fig/http-message-util": "^1.1.2",
         "psr/http-message": "^1.0.1",
-        "zendframework/zend-expressive-router": "^3.0.0alpha1 || ^3.0",
+        "zendframework/zend-expressive-router": "dev-hotfix/options-request",
         "zendframework/zend-psr7bridge": "^0.2.2 || ^1.0.0",
         "zendframework/zend-router": "^3.0.2"
     },
     "require-dev": {
         "malukenho/docheader": "^0.1.6",
-        "phpunit/phpunit": "^6.5.3",
+        "phpunit/phpunit": "^7.0.2",
```

Comment:
Can we please stop with all these bugfix requirements all the time? Why is 7.0.2 needed? Do the tests fail with ^7.0?

Reply:
For all packages we are always using bugfixes version, why we shouldn't do the same for PHPUnit? It's the latest version available, and it doesn't matter too much as it is just testing framework. I wouldn't bother about it at all.

Reply:
I'm with @xtreamwayz here; if you're not updating the requirements for a specific reason, there's no need to bump them typically.

(That said, I won't block this PR based on that; just noting that we shouldn't be bumping requirements on every patch, but only when we really need to.)